syft/generic/tensor.py

```diff
@@ -1,9 +1,8 @@
-import functools
 from typing import List
 import weakref
 
 import syft as sy
-from syft.generic.object import _apply_args
+from syft.generic.object import _apply_args  # noqa: F401
```

Comment:
Does this import have side effects that require it to imported even though it isn't used?